.travis.yml

```diff
@@ -41,6 +41,13 @@ install:
   # install pysat
   - cd /home/travis/build/rstoneback/pysat
   - python setup.py install
+
+# Set up a display for plot tests
```

Comment:
This is really good. I can test my sgp4 plots now. That should help our test coverage.